swiss/src/app/landsraad/landsraad.component.ts

```diff
@@ -103,4 +175,13 @@ export class LandsraadComponent implements OnInit {
     this.resultsShown = event.checked
     this.db.object("landsraad/votingConfig/resultsShown").set(event.checked)
   }
+
+  questionEligibleToDisplay(question: ValueNameExtended, showHiddenQuestions: boolean, filterByRoundId: string) {
+    // questionType available
+    return !!question["questionType"]
```

Comment:
What does `!!` operator do? You might teach me something :)

Reply:
It is boolean representation of variable. So if there is some string is will result in true, for empty string or undefined or null it will result in false. It is quite nice shorthand :)

Reply:
In Kotlin this means potential nullpointerexception crash, so I never let it pass code review :) But it's fine in JS.

Reply:
Ah ok so Kotlin is more strict, well like Java. JS is different in this :)